plugins/blip/basics/components/day/HoverDay.js

```diff
@@ -0,0 +1,53 @@
+var React = require('react');
+var moment = require('moment');
+var cx = require('classnames');
+var _ = require('lodash');
+
+var HoverDay = React.createClass({
+  propTypes: {
+    dayAbbrevMask: React.PropTypes.string.isRequired,
+    data: React.PropTypes.object,
+    date: React.PropTypes.string.isRequired,
+    onHover: React.PropTypes.func.isRequired
+  },
+  getDefaultProps: function() {
+    return {
+      dayAbbrevMask: 'MMM D'
+    };
+  },
+  mouseEnter: function () {
```

Comment:
Maybe pass down `future` too and don't pass up the date when hovering on a future day? The hover state is a bit odd and useless when no data exists...